website/i18n/pt-PT.json

```diff
@@ -499,6 +499,36 @@
       },
       "version-4.2.1/version-4.2.1-talks": {
         "title": "Talks"
+      },
+      "version-4.3.0/version-4.3.0-e2e": {
+        "title": "End to End testing"
+      },
+      "version-4.3.0/version-4.3.0-reverse-proxy": {
+        "title": "Reverse Proxy Setup"
+      },
+      "version-4.3.0/version-4.3.0-who-is-using": {
+        "title": "Who is using Verdaccio?"
+      },
+      "version-4.3.3/version-4.3.3-configuration": {
+        "title": "Configuration File"
+      },
+      "version-4.3.3/version-4.3.3-uplinks": {
+        "title": "Uplinks"
+      },
+      "version-4.3.3/version-4.3.3-who-is-using": {
+        "title": "Who is using Verdaccio?"
```

Comment:
Hm... This is definitely not portuguese 😅 should I merge it anyway @juanpicado ?

Reply:
it fallback to english